README.md

```diff
@@ -136,6 +125,16 @@ To filter these warnings:
 
     tail -f ansible.log | grep DEBUG
 
+Known Issues
+------------
+
+The `template` Ansible modules writes an inventory file and it may complain
+about missing `libselinux-python` which is already installed is most Linux
+distros. If thats the case, simply create a symlink to the selinux directory in
+your virtual environment:
+
+    ln -s /usr/lib64/python2.7/site-packages/selinux/ venv/lib64/python2.7/site-packages/
```

Comment:
What about the other workaround, of creating a virtualenv with `--system-site-packages`

Reply:
Done

Reply:
Added